presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/s3config/S3TableConfigClient.java

```diff
@@ -74,12 +74,14 @@
 
     private final Map<String, KinesisStreamDescription> descriptors = Collections.synchronizedMap(new HashMap<>());
 
+    private final int updateInterval;
     @Inject
     public S3TableConfigClient(
             KinesisConfig connectorConfig,
             KinesisClientProvider clientManager,
-            JsonCodec<KinesisStreamDescription> jsonCodec)
+            JsonCodec<KinesisStreamDescription> jsonCodec, int updateInterval)
```

Comment:
Can we move this to a new line ?

Reply:
Where do we specify this `updateInterval` ? 
Can we configure it ? 
If yes can we maintain in in `KinesisConfig` and fetch it ?

Reply:
yes, cnfigured it via KinesisConfig. Please verify latest commit.